health-services/project-factory/src/server/utils/request.ts

```
@@ -148,7 +149,8 @@ const httpRequest = async (
           errorResponse?.data || { Errors: [{ code: error.message, description: error.stack }] }
         )}`
       );
-      if (retry) {
+      if (retry|| (config.values.autoRetryIfHttpError&&config.values.autoRetryIfHttpError?.includes(errorResponse?.data?.Errors?.[0]?.code))) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)